aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -67,17 +67,22 @@ module aptos_framework::aggregator_v2 {
     }
 
     /// Returns `max_value` exceeding which aggregator overflows.
-    public fun max_value<IntElement: copy + drop>(aggregator: &Aggregator<IntElement>): IntElement {
+    public fun max_value<IntElement: copy + drop>(
+        aggregator: &Aggregator<IntElement>
+    ): IntElement {
         aggregator.max_value
     }
 
     /// Creates new aggregator, with given 'max_value'.
     ///
     /// Currently supported types for IntElement are u64 and u128.
     /// EAGGREGATOR_ELEMENT_TYPE_NOT_SUPPORTED raised if called with a different type.
-    public native fun create_aggregator<IntElement: copy + drop>(max_value: IntElement): Aggregator<IntElement>;
+    public native fun create_aggregator<IntElement: copy + drop>(max_value: IntElement): Aggregator<
+        IntElement>;
```

Comment:
This is a bug?

Reply:
yeah, we created issue about it in https://github.com/movebit/movefmt/issues/16

Reply:
fixed, you can see at https://github.com/movebit/movefmt/issues/16